lib/utils/token.ex

```diff
@@ -12,7 +12,7 @@ defmodule PhxPlatformUtils.Utils.Token do
   end
 
   def get_matching_permissions(conn, allowed_permissions) do
-    allowed = allowed_permissions |> MapSet.new()
+    allowed = allowed_permissions |> MapSet.new() |> MapSet.put("sys:admin")
```

Comment:
We really shouldn't be using "sys:admin" in anywhere.  Lets make a task to ensure we replace this usage with resource/action based permissions.

Reply:
yeah sounds good, we will probably also have to add some permissions in auth0 so the new routes will work properly for non admins, I'll add a task for that too.